content/en/docs/setup/best-practices/certificates.md

```diff
@@ -137,9 +140,14 @@ You must manually configure these administrator account and service accounts:
 | kubelet.conf            | default-auth               | system:node:`<nodeName>` (see note) | system:nodes   |
 | controller-manager.conf | default-controller-manager | system:kube-controller-manager |                |
 | scheduler.conf          | default-scheduler          | system:kube-scheduler          |                |
+| proxier.conf            | default-proxier            | system:kube-proxy              |                |
+
+[3]: any IP or DNS name assigned to your kubelet/kube-proxy host.
 
 {{< note >}}
 The value of `<nodeName>` for `kubelet.conf` **must** match precisely the value of the node name provided by the kubelet as it registers with the apiserver. For further details, read the [Node Authorization](/docs/reference/access-authn-authz/node/).
+
+`kubelet.csr` and `proxier.csr` need **hosts (subjectAltName)** entries that match precisely the value of the node name provided by the kubelet.
```

Comment:
Two things:

1. You mention proxier.csr but only talk about the kubelet.
2. Client identity is not conveyed by subject alternative names. Only the common name is used.